

exec('1+1')
#*ERROR* 30.py:3: 'exec' is not supported
